HOWTO-make-a-release.md

```diff
@@ -243,20 +259,27 @@ The manual for that script is found in `$TOOLS/release-tools/MKRELEASE.md`
 *The changes in this section should be made in your clone of the release
 data repo*
 
-Update the newsflash.txt file.  This normally is one or two lines.  Just
-copy and paste existing announcements making minor changes for the date and
-version number as necessary.  If there is an advisory then ensure you
-include a link to it.
+-   Newsflash *[only for public releases]*
+
+    Update the newsflash.txt file.  This normally is one or two lines.  Just
+    copy and paste existing announcements making minor changes for the date
+    and version number as necessary.  If there is an advisory then ensure
+    you include a link to it.
 
-Update the vulnerabilities.xml file if appropriate.
+-   Security advisory *[both public and premium releases]*
 
-If there is a Security Advisory then copy it into the secadv directory.
+    Update the vulnerabilities.xml file if appropriate.
 
-*Do* send the commits to the reviewer and await their approval.
+    If there is a Security Advisory then copy it into the secadv directory.
 
 Commit your changes, but *do not push* them to the release data repo at this
 stage.  (the release data repo being `git@github.openssl.org:omc/data.git`)
 
+*Do* send the commits to the reviewer and await their approval.  It's
+advisable to use this command to get a copy of those commits:
+
+    git format-patch
```

Comment:
Now this is in the data repo, none of this seems necessary any more. Just create a PR.

Reply:
Oh!  Right, this was from gitolite times...

Reply:
Rewritten